user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_pattern_matching.md

```diff
@@ -26,3 +28,18 @@ If you are viewing a trend graph for a single parameter, and it contains pattern
 
 > [!TIP]
 > See also: [Monitoring of trend patterns](xref:Monitoring_of_trend_patterns)
+
+## Multivariate patterns
+
+From DataMiner 10.3.8/10.4.0 onwards, DataMiner Analytics can automatically recognize recurring patterns in trend graphs showing trend information of multiple parameters. <!-- RN 36454 + 36731 + 36327 -->
+
+When you open a trend graph showing trend information for multiple parameters, you have the option to define a *multivariate trend pattern*.
+
+As with single-parameter patterns (i.e. [univariate patterns](#univariate-patterns)), you can [specify a number of options](xref:Managing_pattern_definitions) in the *Pattern Overview* window. <!-- RN 35010 -->
+
+If you are viewing a trend graph where all parameters involved in the multivariate pattern are loaded and multivariate patterns have been detected, these patterns will be highlighted in orange when you hover the mouse pointer over the button representing a pattern, or if the option *Expand tags* is selected in the right-click menu.
+
+> [!NOTE]
+> You do not need to have multiple parameters displayed in the trend graph to see the detected pattern occurrences.
```

Comment:
Is this more accurate, @VeerleLedoux?

Reply:
I suggest to omit the note and changing it into "If you are viewing a trend graph where one or more of the parameters involved in the multivariate pattern are loaded and multivariate patterns have been detected, these patterns will be highlighted in orange when you hover the mouse pointer over the button representing a pattern, or if the option *Expand tags* is selected in the right-click menu."

Reply:
Implemented in [commit e905b82](https://github.com/SkylineCommunications/dataminer-docs/pull/1699/commits/e905b8213d4b28cecd02a2f186ee38905e44e582). Is this now OK to merge?